### PR TITLE
Added API extension for recovery and test certificates 

### PR DESCRIPTION
**Pull request by oliver-steinbrecher**

### CODE REVIEW

File: vaccination-certify-api.yaml [hunk 1]
```diff
@@ -64,9 +64,18 @@ paths:
...
               $ref: '#/components/schemas/VaccinationRequest'
+          application/vaccination.certificate.v1+json:
+            schema:
+              $ref: '#/components/schemas/VaccinationRequest'
+          application/recovery.certificate.v1+json:
+            schema:
+              $ref: '#/components/schemas/RecoveryRequest'
+          application/test.certificate.v1+json:
+            schema:
+              $ref: '#/components/schemas/TestRequest'
```
Comment (oliver-steinbrecher) [inline vaccination-certify-api.yaml:78]: Added semantic versioning and explicit content-typse

**Review by thinkberg — COMMENTED** [review]
This was a mistake. Its already in the main branch.

**Comment by thinkberg** [comment]
Check mal bitte.

File: vaccination-certify-api.yaml [hunk 2]
```diff
@@ -61,12 +61,78 @@ paths:
         **Applications:** vaccination center software, patient information systems, integrators
       requestBody:
         description: >-
-          Vaccination information. The value-sets are defined in the
+          Digital Green Certificate request information. The value-sets are defined in the
           [EU eHealthNetwork: Value Sets for Digital Green Certificates. version 1.0](https://ec.europa.eu/health/sites/default/files/ehealth/docs/digital-green-certificates_dt-specifications_en.pdf).
         content:
-          application/json:
```
Comment (oliver-steinbrecher) [inline vaccination-certify-api.yaml:67]: application/json is required to support backwards compatibility

**Pull request merged**